fs.go

```diff
@@ -16,6 +15,13 @@ import (
 	"github.com/pkg/errors"
 )
 
+// Marshaler is the interface implemented by types that
+// can marshal themselves into valid TOML.
+// TODO(carolynvs) Consider adding this to go-toml?
+type Marshaler interface {
```

Comment:
I ended up needing this to use reflection for the `LockDiff`. Here's the go-toml PR: https://github.com/pelletier/go-toml/pull/151